frontend/src/components/Pagination.js

```diff
@@ -6,28 +6,25 @@ import { ButtonGroup, Button } from "react-bootstrap";
 const Pagination = (props) => {
   let renderPaginationBox = (content, pageIndex) => (
     <Button
-      key={`pagination-box-${pageIndex}`}
+      key={`pagination-box-${content}`}
       className="PaginationBox"
-      variant="secondary"
+      variant={(pageIndex === props.currentPageIndex ? "primary" : "secondary")}
```

Comment:
What happens if you get to the last page?

Reply:
The button of last page will be showed as primary button.

Reply:
If you are on the last page, should the last page button be disabled?

Reply:
If it's on the last page, the "Next" and "Last" button will be hidden.